rmtk/vulnerability/Convolution_module/Convolution.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+import os
+import numpy
+import math
+import csv
+import matplotlib.pyplot as plt
+from rmtk.vulnerability.common import utils
+from scipy.stats import lognorm
+
+def calculate_convolution(hazard_curve,fragility_model,return_period,damage_state,save,plot,typology):
+#This function calculates the annual probability of collapse given a hazard 
+#curve and a fragility model
+
+   rate_exceed = []
```

Comment:
Please check the indentation for all functions in this module ([PEP 8 link for reference](https://www.python.org/dev/peps/pep-0008/#indentation)).